test/conformance/tests/test_sriov_operator.go

```diff
@@ -1724,82 +1756,152 @@ var _ = Describe("[sriov] operator", func() {
-							NetworkNamespace: namespaces.Test,
-						}}
-
-					// We need this to be able to run the connectivity checks on Mellanox cards
-					if intf.DeviceID == "1015" {
-						sriovNetwork.Spec.SpoofChk = off
-					}
-
-					err = clients.Create(context.Background(), sriovNetwork)
-
-					Expect(err).ToNot(HaveOccurred())
 
-					Eventually(func() error {
-						netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
-						return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: "test-mtuvolnetwork", Namespace: namespaces.Test}, netAttDef)
-					}, (10+snoTimeoutMultiplier*110)*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
-
-				})
+						sriovNetwork := &sriovv1.SriovNetwork{
+							ObjectMeta: metav1.ObjectMeta{
+								Name:      "test-mtuvolnetwork",
+								Namespace: operatorNamespace,
+							},
+							Spec: sriovv1.SriovNetworkSpec{
+								ResourceName:     resourceName,
+								IPAM:             `{"type":"host-local","subnet":"10.10.10.0/24","rangeStart":"10.10.10.171","rangeEnd":"10.10.10.181","routes":[{"dst":"0.0.0.0/0"}],"gateway":"10.10.10.1"}`,
+								NetworkNamespace: namespaces.Test,
+							}}
+
+						// We need this to be able to run the connectivity checks on Mellanox cards
+						if intf.DeviceID == "1015" {
+							sriovNetwork.Spec.SpoofChk = off
+						}
 
-				// 27662
-				It("Should support jumbo frames", func() {
-					podDefinition := pod.DefineWithNetworks([]string{"test-mtuvolnetwork"})
-					firstPod, err := clients.Pods(namespaces.Test).Create(context.Background(), podDefinition, metav1.CreateOptions{})
-					Expect(err).ToNot(HaveOccurred())
+						err = clients.Create(context.Background(), sriovNetwork)
 
-					Eventually(func() corev1.PodPhase {
-						firstPod, _ = clients.Pods(namespaces.Test).Get(context.Background(), firstPod.Name, metav1.GetOptions{})
-						return firstPod.Status.Phase
-					}, 3*time.Minute, time.Second).Should(Equal(corev1.PodRunning))
+						Expect(err).ToNot(HaveOccurred())
 
-					By("Checking MTU in pod network status annotation")
-					networkStatusJSON, exist := firstPod.Annotations["k8s.v1.cni.cncf.io/network-status"]
-					Expect(exist).To(BeTrue())
-					Expect(networkStatusJSON).To(ContainSubstring("\"mtu\": 9000"))
+						Eventually(func() error {
+							netAttDef := &netattdefv1.NetworkAttachmentDefinition{}
+							return clients.Get(context.Background(), runtimeclient.ObjectKey{Name: "test-mtuvolnetwork", Namespace: namespaces.Test}, netAttDef)
+						}, (10+snoTimeoutMultiplier*110)*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 
-					var stdout, stderr string
-					Eventually(func() error {
-						stdout, stderr, err = pod.ExecCommand(clients, firstPod, "ip", "link", "show", "net1")
-						if stdout == "" {
-							return fmt.Errorf("empty response from pod exec")
+						// update the interface
```

Comment:
please, extract a function like 



```suggestion
						intf = getInterfaceFromNodeStateByPciAddress(node, intf.PciAddr)
						
						func getInterfaceFromNodeStateByPciAddress(node, pciAddress string) {
						    nodeState := &sriovv1.SriovNetworkNodeState{}
						    err = clients.Get(context.Background(), runtimeclient.ObjectKey{Name: node, Namespace: operatorNamespace}, nodeState)
						    Expect(err).ToNot(HaveOccurred())
						    found := false
						    for _, state := range nodeState.Status.Interfaces {
							    if state.PciAddress == intf.PciAddress {
								    intf = state.DeepCopy()
								    found = true
								    break
							    }
						    }
						    Expect(found).To(BeTrue())
						}
```

WDYT?